<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>CVE Feed for ForgeRock -- LDAP Connector</title>
    <link>https://raw.githubusercontent.com/deepseas/cvelistV5monitor/main/feeds/forgerock/ldap%20connector.rss</link>
    <description>The latest CVEs for ForgeRock -- LDAP Connector</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Tue, 17 Sep 2024 14:22:42 +0000</lastBuildDate>
    <ttl>60</ttl>
    <item>
      <title>CVE-2022-0143|2024-09-17T03:02:14.548Z -- forgerock -- ldap%20connector
</title>
      <link>https://www.cve.org/CVERecord?id=CVE-2022-0143</link>
      <description>When the LDAP connector is started with StartTLS configured, unauthenticated access is granted. This issue affects: all versions of the LDAP connector prior to 1.5.20.9. The LDAP connector is bundled with Identity Management (IDM) and Remote Connector Server (RCS)</description>
      <guid isPermaLink="false">CVE-2022-0143|2024-09-17T03:02:14.548Z</guid>
      <pubDate>Mon, 19 Sep 2022 21:15:51 +0000</pubDate>
    </item>
  </channel>
</rss>
